click-to-reveal/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
```

Comment:
```suggestion
<html lang="en">
```

I ran your HTML through a validator and it suggested adding `lang` to your `<html>` tag.